docs/content/guides/accessibility/accessibility.md

```diff
@@ -0,0 +1,185 @@
+---
+id: o4qhm1bg
+title: Handsontable - Accessibility
+metaTitle: Accessibility - JavaScript Data Grid | Handsontable
+description: Learn about Handsontable's accessibility features.
+permalink: /accessibility
+canonicalUrl: /accessibility
+tags:
+  - accessibility
+  - a11y
+  - aria
+  - jaws
+  - nvda
+  - voiceover
+  - wcag
+  - section 508
+  - ada
+  - compliance
+  - vpat
+  - acr
+react:
+  id: x82phf34
+  metaTitle: Accessibility - React Data Grid | Handsontable
+searchCategory: Guides
+---
+[[toc]]
+
+## Overview
+
+Handsontable offers comprehensive accessibility support, including built-in keyboard navigation in line with international standards. Our design ensures that individuals with disabilities can fully engage with Handsontable.
```

Comment:
I am unsure about this paragraph, I think the way we use "Our design" is incorrect. I am thinking something along the lines of

"We made Handsontable accessible to individuals with disabilities due to our deep belief in the principle “design for all”. Web applications should be as inclusive as possible."

or perhaps we could rephrase the existing to something like "Handsontable offers comprehensive accessibility support in line with international standards. Due to our deep belief in the principle “design for all”. Web applications should be as inclusive as possible."

I think we should remove the mention of keyboard navigation in this paragraph because we are more than just that and we would have to include more in the paragraph to do it justice.

Reply:
changed